packages/@okta/vuepress-site/docs/guides/revoke-tokens/overview/index.md

```diff
@@ -4,7 +4,15 @@ title: Overview
 
 If you would like to disable an access or refresh token, simply send a request to the `/revoke` endpoint for the appropriate Authorization Server.
 
-This example makes a request to revoke an access token issued by the Org Authorization Server. The URL of the Authorization Server for your request might be different than this example. See [Authorization Servers](/docs/concepts/auth-servers/#available-authorization-server-types) for more information.
+This example makes a request to revoke an access token issued by the Org Authorization Server. The URL of the Authorization Server for your request might be different than this example.
+
+**Examples**
+
+The URL for an Org Authorization Server: `https://${yourOktaDomain}/oauth2/`
```

Comment:
I was thinking this example would be under the section where we describe the baseUrl and not specifically for token revocation